inc/languages/english/moderation.lang.php

```diff
@@ -159,7 +159,7 @@
 $l['purgespammer_modlog'] = "Purged Spammer";
 
 $l['error_invalidpm'] = "Invalid PM";
-$l['error_nomergeposts'] = "You need to select one or more posts to be able to merge posts together.";
+$l['error_nomergeposts'] = "You need to select atleast two or more posts to be able to merge posts together.";
```

Comment:
Minor issue, space missing.
```suggestion
$l['error_nomergeposts'] = "You need to select at least two or more posts to be able to merge posts together.";
```

Reply:
Done.